tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend-test-ci-pipeline.yml

```diff
@@ -6,7 +6,7 @@ trigger: none
 jobs:
 - job: Onnxruntime_Linux_GPU_Training_FrontEnd
 
-  timeoutInMinutes: 120
+  timeoutInMinutes: 240
```

Comment:
Will the new test add 2h to the total? That is half day of waiting for a single pipeline

Reply:
Each multiple choice test takes about 60 mins so 240 is a safe number here. I am working on parallel training so hopefully when we have parallel runs it will take less time. On the other hand, we shall not worry about time limit here because it is a nightly test. With more tests being added, I expect the limit will be raised higher.